community-operators/knative-kafka-operator/0.12.1/knative-kafka-operator.v0.12.1.clusterserviceversion-v0.12.1.yaml

```diff
@@ -0,0 +1,590 @@
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - deployments/status
+          verbs:
+          - get
+          - list
+          - watch
+        serviceAccountName: kafka-ch-controller
+      - rules:
+        - apiGroups:
+          - messaging.knative.dev
+          resources:
+          - kafkachannels
+          verbs:
+          - get
+          - list
+          - watch
+          - patch
+        - apiGroups:
+          - messaging.knative.dev
+          resources:
+          - kafkachannels/status
+          verbs:
+          - get
+          - update
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - get
+          - list
+          - watch
+          - patch
+          - create
+        - apiGroups:
+          - messaging.knative.dev
+          resources:
+          - kafkachannels
+          verbs:
+          - get
+          - list
+          - watch
+          - patch
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: kafka-ch-dispatcher
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - admissionregistration.k8s.io
+          resources:
+          - mutatingwebhookconfigurations
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+          - patch
+          - watch
+        - apiGroups:
+          - messaging.knative.dev
+          resources:
+          - kafkachannels
+          - kafkachannels/status
+          verbs:
+          - get
+          - list
+          - watch
+        serviceAccountName: kafka-webhook
+      - rules:
+        - apiGroups:
+          - sources.eventing.knative.dev
+          resources:
+          - kafkasources
+          verbs:
+          - get
+          - list
+          - watch
+        serviceAccountName: eventing-contrib-kafka-source-observer
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: knative-kafka-operator
+      deployments:
```

Comment:
In previous version (0.11.2), the only deployment in the CSV was the knative-kafka-operator ... the operator then installed the kafka-ch-dispatcher , controller and webhook after creating the KnativeEventingKafka CR.  Having these in CSV will mean these will be created in the namespace of the Subscription.  Was this change deliberate?

Reply:
I think the deployments were added in the CSV by mistake.

Reply:
@aliok can you follow up on this?  Since you did the actual Operator update, there might be some diff between your work, and my PR to the hub.

I can also close this now, and have you PR the ultimate solution